tests/infra/github.py

```diff
@@ -163,8 +163,8 @@ def get_latest_dev_tag(self):
         return self._filter_released_tags(dev_tags)[0]
 
     def get_tags_for_major_version(self, major_version=None):
-        version_re = f"{major_version}\." if major_version else ""
-        tag_re = f"^{TAG_RELEASE_PREFIX}{version_re}([.\d+]+)(-rc.*|)$"
+        version_re = rf"{major_version}\." if major_version else ""
```

Comment:
@jumaffre: These changes are just for the linting issues introduced in #3512, that are also solved by #3515 - I'll take those once that's merged.